service/handlers.go

```diff
@@ -177,6 +177,8 @@ func handlePoolPost(c *Core, w *ResponseWriter, r *Request) {
 		w.Error(err)
 		return
 	}
+
+	c.publishEvent("new-pool", pool.ID.String())
```

Comment:
Shouldn't there also be an event for put pool (ie update name) and delete pool?

Reply:
Also I don't think its good that the payload is an inconsistent content type. I would just stick with json.